Dockerfile

```diff
@@ -7,6 +7,8 @@ COPY package*.json ./
 
 RUN npm install
 
```

Comment:
I had to do this since running `docker-compose up` was looking for this folder when installing dependencies